tests/src/data/use-cases/pet/db-appoint-specie.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { type LoadSpecieByNameRepository } from '@/data/protocols'
+import { DbAppointSpecie } from '@/data/use-cases'
+import { type AppointSpecie } from '@/domain/use-cases'
+import { makeFakeSpecieRepository } from '@/tests/utils'
+
+interface SutTypes {
+  sut: DbAppointSpecie
+  specieRepositoryStub: LoadSpecieByNameRepository
+}
+
+const makeSut = (): SutTypes => {
+  const specieRepositoryStub = makeFakeSpecieRepository()
+  const dependencies: AppointSpecie.Dependencies = {
+    specieRepository: specieRepositoryStub
+  }
+  const sut = new DbAppointSpecie(dependencies)
+
+  return {
+    sut,
+    specieRepositoryStub
+  }
+}
+
+describe('DbAppointOtherSpecie Use Case', () => {
+  const specieName: AppointSpecie.Params = 'any_name'
+
+  describe('LoadSpecieByName', () => {
```

Comment:
Here it should be `describe("specieRepository")` since the sut dependency is specieRepository